www/js/chart-api-test.js

```diff
@@ -124,8 +170,21 @@ function redraw() {
       .then(function (response) {
         processed = processApiMempoolDataForChart(response.data)
         draw(processed)
+        updateMainPage(processed)
         redraw()
       });
   }, 60000);
+}
+
+function updateMainPage(processed) {
+  const total = [Object.values(processed.rowData[1]).reduce((a, b) => a + b, 0)]
+  document.getElementById('total-transactions').innerHTML = total + ' unconfirmed transasctions (23 MB).' // TODO: get size of mempool from API
+
+  timeSinceLastUpdate = 0
+  document.getElementById('last-update').innerHTML = 'last updated ' + timeSinceLastUpdate + ' minutes ago'
```

Comment:
I guess just updating the `timeSinceLastUpdate` is cleaner in case we change the text sometime in the future.